src/components/AssetManagementPanel/AssetManagementForm/FormVariants/SurrenderForm/SurrenderForm.tsx

```diff
@@ -41,15 +41,14 @@ export const SurrenderForm = ({
   useEffect(() => {
     if (isConfirmed) {
       showOverlay(showDocumentTransferMessage(MessageTitle.SURRENDER_DOCUMENT_SUCCESS, { isSuccess: true }));
+      onBack(); // onSetFormAction to none
```

Comment:
lol dont have to add the comment everywhere, if you really want to, change the name of onBack to something clearer.

Good variable naming > Comments to supplement bad naming.

Reply:
I think i saw something like that on top also, can either remove the comment or change the variable name.